packages/service-content/src/lib/resources/queries/get-newsletter.ts

```diff
@@ -0,0 +1,32 @@
+import { sql } from '@blms/database';
+import type { JoinedNewsletter } from '@blms/types';
+
+export const getNewsletterQuery = (id: number, language?: string) => {
```

Comment:
Changes to make there related to expected changes in drizzle schema (use language in localized table instead)